aesara/tensor/math.py

```diff
@@ -1800,13 +1800,13 @@ def true_divide(a, b):
 
 
 @scalar_elemwise
-def int_div(a, b):
+def floor_divide(a, b):
     """elementwise [floor] division (inverse of multiplication)"""
     # see decorator for function body
 
 
-# floor_div and int_div are the same thing
-floor_div = int_div
+# floor_divide and int_div are the same thing
+int_div = floor_divide
```

Comment:
```suggestion
```
I don't think these references are necessary when an entry for the deprecated object is added to `DEPRECATED_NAMES` below.